### ISSUE TITLE
BotBits client crashes when orange switch is placed.

### ISSUE THREAD

**Issue by mateon1** [Yonom/BotBits#17]
When an orange switch is placed, an ArgumentException is thrown internally in BotBits.dll (That block doesn't accept any extra parameters!)

All I can copy to clipboard is:

```
System.ArgumentException was unhandled
Message: An unhandled exception of type 'System.ArgumentException' occurred in BotBits.dll
Additional information: That block doesn't accept any extra parameters!
```

I can't get any stack trace to show up.

**Comment by Yonom** [comment]
Sorry I saw this just now. Thanks for the report, I'll try to get it fixed in a few hours.

**Comment by Yonom** [comment]
I forgot to add the orange blocks to Foreground, How silly!

Fixed in v211.4.4.
